timed/permissions.py

```diff
@@ -99,21 +112,35 @@ def has_permission(self, request, view):
         if not super().has_permission(request, view):  # pragma: no cover
             return False
 
-        if request.method not in SAFE_METHODS:
-            return request.user.reviews.exists()
-
-        return True
+        if request.method in ["PATCH", "GET", "POST"]:
+            return True
```

Comment:
No need to check for the access method. This is done by combining the permission with for example `IsNotDelete` etc.